src/carousel/carousel.ts

```diff
@@ -43,7 +45,7 @@ export class NgbSlide {
   },
   template: `
     <ol class="carousel-indicators">
-      <li *ngFor="let slide of slides" [id]="slide.id" [class.active]="slide.id === activeId" (click)="cycleToSelected(slide.id)"></li>
+      <li *ngFor="let slide of slides" [id]="slide.id" [class.active]="slide.id === activeId" (click)="select(slide.id)"></li>
```

Comment:
This changes the behaviour in subtle way: now we would reset timer on click. Was it intentional? If so we should add a test for it.